aten/src/ATen/native/cuda/RNN.cu

```diff
@@ -82,7 +82,7 @@ namespace kernel {
 
 template <typename scalar_t, typename accscalar_t, typename index_type, int indexing_kind>
 #if __CUDA_ARCH__ >= 350 || defined __HIP_PLATFORM_HCC__
-C10_LAUNCH_BOUNDS_2(512, 4)
```

Comment:
can you define constexpr max threads in this file for forward and backward to make sure that kernel launch bounds and arguments to `getLaunchConfig` call don't diverge?

Reply:
I defined a constexpr MAX_NUM_THREADS = 256, but lstm_cell_forward needs 128 to fully get rid of lmem, so I just used MAX_NUM_THREADS / 2. Hope that's ok, alternatively I can define a separate max threads variable for specifically lstm_cell_forward, since the others all use 256.